pkg/security/security.go

```diff
@@ -34,6 +34,11 @@ const (
 
 	// DefaultRootCertFilePath is the well-known path for an existing root certificate file
 	DefaultRootCertFilePath = "./etc/certs/root-cert.pem"
+
+	// Platforms
+	K8S  = "Kubernetes"
+	GCE  = "GoogleComputeEngine"
```

Comment:
What about all other platforms we support ? Are they mock ? 

If you really want 'platform' - please write a RFC, get it reviewed by usability, install, env - and likely TOC.

Reply:
Reading a file containing an OIDC token is not "k8s platform". It works on all platforms, and there is nothing in the code that will make it k8s specific.